features/build/buildlogic.feature

```diff
@@ -458,3 +458,45 @@ Feature: buildlogic.feature
       | sample-pipeline.*Cancelled |
       | sample-pipeline.*Complete  |
     """
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-40366
+  @admin
+  Scenario: Mirroring built image doesn't degrade scheme2 ,keep consistent SHA's	
+    Given I have a project
+    Given I save a htpasswd registry auth to the :combare_dockercfg clipboard
```

Comment:
@xiuwang is this a typo?  `:combare_dockercfg`?

Reply:
@pruan-rht You caught me 👍  I have updated it, please help review again, thanks